haiku-apps/calendar/calendar-1.0.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="A native Calendar application for Haiku"
+DESCRIPTION="A native Calendar application for Haiku
+Features:
+Create, modify and delete events.
+Generate notifications for events.
+Day Calendar view.
+Event categorization.
+Set 'All day' long events.
+Fetching events from Google Calendar using Google Calendar API.
+SQLite backend for storing events.
+Setting preferences like 'First day of week', 'Display week number in \
+Calendar'.
+App localization: DateTime strings are localised and updates with locale \
+preferences change but GUI string still needs to be localized."
+HOMEPAGE="https://www.gnu.org/software/libcdio/"
+COPYRIGHT="2007-2017 Rocky Bernstein and Herbert Valerio Riedel"
```

Comment:
Where did this info come from? The native Calendar app for Haiku was written during GSoC 2017.

Reply:
Jikes ;)

Reply:
What would be the best to use here? COPYRIGHT="2017 GSoC"?

Reply:
2017 Akshay Agarwal
2017-2018 Haiku Archives Team